field-agent/src/components/FieldAgents.js

```diff
@@ -0,0 +1,420 @@
+
+                            <div className="field">
+                                <label for="firstName">First Name:</label>
+                                <input id="firstName" name="firstName" type="text" onChange={firstNameOnChangeHandler}></input>
+                            </div>
+
+                            <div className="field">
+                                <label for="middleName">Middle Name:</label>
+                                <input id="middleName" name="middleName" type="text" onChange={middleNameOnChangeHandler}></input>
+                            </div>
+
+                            <div className="field">
+                                <label for="lastName">Last Name:</label>
+                                <input id="lastName" name="lastName" type="text" onChange={lastNameOnChangeHandler}></input>
+                            </div>
+
+                            <div className="field">
+                                <label for="dob">DOB:</label>
+                                <input id="dob" name="dob" type="date" onChange={dobOnChangeHandler}></input>
+                            </div>
+
+                            <div className="field">
+                                <label for="heightInInches">Height in Inches:</label>
+                                <input id="heightInInches" name="heightInInches" type="number" min="0" max="200" onChange={heightInInchesOnChangeHandler}></input>
+                            </div>
+
+                            <button><i className="fas fa-plus-circle"></i> Add Agent</button>
+
+                        </form>
+
+                        <button type="button" onClick={agentListClickHandler} >Cancel</button>
+
+                    </main>
+                    <footer>
+                        <section>
+                            <h2>Legal Notices</h2>
+                            <p>Copyright &copy; 2021 Field Agent Inc. All rights reserved | <a href="#">Privacy Policy</a></p>
+                        </section>
+                    </footer>
+                </div>
+            ) : null}
+
+            {currentView === 'Edit' ? (
+                <div class="container">
+                    <header>
+                        <h1>Field Agent</h1>
+                        <div>
+                            Welcome username! | <a href="#">Logout</a> | <a href="login.html">Login</a> | <a
+                                href="register.html">Register</a>
+                        </div>
+                    </header>
+                    <nav>
+                        <div>
+                            <button type="button" onClick={homeClickHandler} >Home</button> | <a href="#">Agencies</a> | <a href="#">Locations</a> |
+                            <button type="button" onClick={agentListClickHandler}>Agents</button> | <a href="#">Missions</a>
+                        </div>
+                    </nav>
+                    <main>
```

Comment:
I agree with Clint on this. Seems like you should be able to place in the conditional logic inside the `<main>` tag and not have to repeat your header, navigation, and footer.